agent/checkpoint.cpp

```diff
@@ -139,7 +139,7 @@ void Checkpoint::addComponentEvent(ComponentEvent *event)
 }
 
 
-void Checkpoint::copy(Checkpoint &checkpoint, std::set<std::string> *filterSet)
+void Checkpoint::copy(Checkpoint const &checkpoint, std::set<std::string> *filterSet)
```

Comment:
I just realised that I put the 'const' modifier after the type. This is our company standard (and C++) but the rest of the project has 'const {type}' style. So these changes are inconsistent. I can create another commit to rectify them all to some agreed standard later if required.